packages/shared/src/components/ProfilePicture.tsx

```diff
@@ -11,7 +11,8 @@ export type ProfileImageSize =
   | 'large'
   | 'xlarge'
   | 'xxlarge'
-  | 'xxxlarge';
+  | 'xxxlarge'
+  | 'profile';
```

Comment:
Rather check with Tomer, I'd like to keep things uniform

Reply:
Agree with that. Awaiting response now 🚢